docker-compose.yml

```diff
@@ -48,7 +48,10 @@ services:
       MONGODB_URL: mongodb://mongodb:27017
       SERVER_SHOW_ERRORS: "true"
       SRV_HORIZON_URL: http://mock-horizon:3000
-      SRV_NOTIFY_URL: http://mock-notify:3000
+#      SRV_NOTIFY_BASE_URL: http://mock-notify:3000
```

Comment:
I think we need these in docker-compose don't we? I've removed it from the Helm values in my PR

Reply:
I have updated the project's README.md to o explain the required and optional environment variables. The values to use the mock Notify service are included as defaults in the docker-compose.yml file.